packages/arb-token-bridge-ui/src/util/ExponentialBackoffUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function wait(ms: number) {
```

Comment:
Not super sure about the filename since `wait` is generic, doesn't have much to do with exponential wait-time, and might be hard to discover later. 
Unless we plan on adding a lot more related functions here, I guess just having `wait` from `CommonUtils` would suffice?

Reply:
Will be taken up in next PR.